### PR TITLE
Add token from Factories World

### PR DESCRIPTION
**Pull request by Harko999**
## How to add token?

- [ ] Add metadata to [`tokens.json`](tokens.json)

```json
{
    "name": "<NAME>",
    "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "symbol": "<SYMBOL>",
    "account": "<CONTRACT NAME>",
    "chain": "eos"
}
```

- [ ] Add token logo `*.png` to [`./logos`](./logos) folder

### CODE REVIEW

File: tokens.json [hunk 1]
```diff
@@ -4822,5 +4822,37 @@
...
+        "name": "Factories World Energy",
+        "logo": "https://github.com/Harko999/eos-airdrops/blob/master/logos/FAWE.png",
+        "logo_lg": "https://github.com/Harko999/eos-airdrops/blob/master/logos/FAWE.png",
+        "symbol": "FAWE",
+        "account": "factorieswrd",
+        "chain": "wax"
+    },
+    {
+        "name": "Factories World Gold",
+        "logo": "https://github.com/Harko999/eos-airdrops/blob/master/logos/FAWG.png",
```
Comment (DenisCarriere) [inline tokens.json:4852]: Follow the URL schema, it's incorrect again:

```
https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png
```

**Review by DenisCarriere — CHANGES_REQUESTED** [review]

**Comment by Harko999** [comment]
How can I make correct link?

сб, 8 окт. 2022 г. в 18:11, Denis ***@***.***>:

**Comment by DenisCarriere** [comment]
> How can I make correct link? сб, 8 окт. 2022 г. в 18:11, Denis ***@***.***>:
> […](#)

Edit the url path to match

**Comment by Harko999** [comment]
ready


сб, 8 окт. 2022 г. в 23:26, Denis ***@***.***>:

**Comment by Harko999** [comment]
Hi eoscafe/eos-airdrops,

I'm using a new tool called Signal that helps founders find the right Investors and helps Investors attract the right deals. A lot of what powers that is who you're connected to. If we connect, it'll help both of us manage our Investor contacts better.

Please confirm our connection: http://signalmail-tracking.nfx.com/ls/click?upn=ojRnCyOQslQVr-2FL3NgSUpZzxrdEAcoYAHVA34eZvZ-2B9oymYhW-2FSKEOsgQMYroBlWsGyQI9FlHkJRrJkJ-2FRLXh97Qj76DLT7Mf-2Fswaygb1v7lx6IESXd5vNv8F65Eebun41n37mmonBlYHK7Tse01UH50S1v-2BaHMdUf0TpxoKq5eo5K0jr-2FSCycFeCVIn6QjrKkCgmTueZTT-2FQVUi2sM8A1lM4V2ejKxq-2B9LvlCHhNmMBHSB4yvvtz7EdU94epImpv4sfRowmq-2Bgz4ge0gynNWQ-3D-3Du2j5_U31jt5HgHBh7k3Grc5s4lzB8gM2J69ke-2BrdQgbX37mEe8BKcf6t4TWwjui8fl2kvnzxa9FM8FnTx-2FD9XDnyqJ6XSqS39snJWeN8j69ePOSI90KwAQND03rtr4Yr8w9GXy56FHHAif-2FS66zciwnAIK7x5x7oquyMtx6pyiRClc-2BMmrBq4m3sAFSyXFFRJzSDTHFpj-2B0puRYIiE-2Fj-2FGUW10iwUZO9BYf84T247jXcXp0jAoRQBJObGy-2BUwN1Ul13kMwGijlTIo-2FrlTdY3PzfT9lbHmwJaSYCpmxfBLLH-2B5REsXE2sXn8fhPWLVBdX13xbN

Best,
Maksim

**Comment by Harko999** [comment]
Hi eoscafe/eos-airdrops,

I'm using a new tool called Signal that helps founders find the right Investors and helps Investors attract the right deals. A lot of what powers that is who you're connected to. If we connect, it'll help both of us manage our Investor contacts better.

Please confirm our connection: http://signalmail-tracking.nfx.com/ls/click?upn=ojRnCyOQslQVr-2FL3NgSUpZzxrdEAcoYAHVA34eZvZ-2B9oymYhW-2FSKEOsgQMYroBlWsGyQI9FlHkJRrJkJ-2FRLXh97Qj76DLT7Mf-2Fswaygb1v7lx6IESXd5vNv8F65Eebun6NkzRdzkxoQCftwaUbJaTw-2B4NuwRek3WWOD-2Fo30CzMR2GOtU2zslB6r-2BEo4Eub4zahAz7NaZfbfzvDHZNDXc7RSC-2BMWML4gwyYjx-2FihphvBJCAkY2JFOgIgxZPDpYTahqBGGqdhj51YsvQ2jWYGzug-3D-3DemyJ_U31jt5HgHBh7k3Grc5s4l6qS4a2s2DPW6Hj1Ifoi4FEAsLd5QQqzNbn-2FhqfZB5sQBK-2B0sVE-2FdRcyiNjTxOq3h0Bjq-2FAoWYCoQuZrY19A6RuGejlWu8dMFWr4AL4ewlrponXQvmyRtQXuCBDGq50GI0aD2CeBPHyJBwu6iynixM4-2BezGG0VM5cM2o-2B0zHF0X12FHN7L0axkvTCYZABWuW0vYssJfhYlU-2BKlm17Kz-2Bb9mcOG-2BEvzRZNcm3qUxO7dPB2uq3DxqLPNxOV0tQcYAM88BUzntL3ebJH2e-2BfjF9NYmt6nKnakwnAF4U7RFEdHsU

Best,
Maksim

**Pull request merged**